association_backend/.flake8

```diff
@@ -0,0 +1,4 @@
+[flake8]
```

Comment:
is this a symbolic link to the .flake8 that is in the root?

Reply:
Now yes :simple_smile: